tests/cypress/e2e/form/editor/conditions.cy.js

```diff
@@ -823,34 +827,39 @@ describe ('Conditions', () => {
 
         // Init test question on which we will add our conditions.
         addQuestion('Test subject');
-        getAndFocusQuestion('Test subject').within(() => {
-            initVisibilityConfiguration();
-            setConditionStrategy('Visible if...');
-        });
-        closeVisibilityConfiguration();
 
-        // Text condition
+        // Create a question for each conditions type
```

Comment:
Note: I've reorganize this tests to add all the questions at the start, then all the conditions.

This is faster than doing question -> condition -> question -> ... as we don't move around the UI as much.